<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="todayInfo_groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>125</height>
    </rect>
   </property>
   <property name="title">
    <string>当前天气</string>
   </property>
   <widget class="QLabel" name="dangqian_wendu_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>16</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="dangqian_tianqi_img_label">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>14</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="dangqian_min_maxwendu_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dangqian_kongqizhiliang_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dangqian_date_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dangqian_kongqizhiliang_img_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Calibri&quot;;


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="currCity_label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>35</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>上海</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="todayInfo_groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>201</width>
     <height>125</height>
    </rect>
   </property>
   <property name="title">
    <string>其它信息</string>
   </property>
   <widget class="QLabel" name="fangshaizhishu_label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fangshaizhishu_label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="liangshaizhishu_label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="liangshaizhishu_label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="yongdongzhishu_label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="yongdongzhishu_label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>45</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="xichezhishu_label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="xichezhishu_label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>95</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QChartView" name="chart_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>287</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QComboBox" name="city_comboBox_p">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="city_comboBox_c">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="city_comboBox_a">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="showSetting_pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>45</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>45</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_City">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currCityID_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reWwather_pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>45</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setting">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
